lib/extension.js

```diff
@@ -1,11 +1,11 @@
 const path = require('path');
 
-const EXTRE = /^[.]?[^.]+([.].*)$/;
+const EXTRE = /(\.[^.]+)+$/;
```

Comment:
based on some logging i'm doing against this branch, it seems this regexp will cause problems